cardstock/uiButton.py

```diff
@@ -50,7 +50,7 @@ def GetCursor(self):
 
     def OnPropertyChanged(self, model, key):
         super().OnPropertyChanged(model, key)
-        if key == "title":
+        if key == "title" or key == "fill_color" or key == "text_color":
```

Comment:
For testing against multiple values, I've been using:
```suggestion
        if key in ["title", "fill_color", "text_color"]:
```